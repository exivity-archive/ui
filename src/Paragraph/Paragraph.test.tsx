import React from 'react'
import renderer from 'react-test-renderer'
import { Paragraph } from '.'

test('renders Paragraph without props', () => {
  const component = renderer.create(<Paragraph />)
  expect(component.toJSON()).toMatchSnapshot()
})
