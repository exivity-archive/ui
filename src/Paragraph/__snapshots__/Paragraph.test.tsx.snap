// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Paragraph without props 1`] = `
.c0 {
  font-family: 'Fira Sans',sans-serif;
  font-weight: normal;
  font-size: 16px;
  color: #222;
  line-height: 1.5;
  color: green;
}

<div
  className="c0"
/>
`;
